### PR TITLE
W3HW6 tests for model code

### PR DESCRIPTION
**Pull request by yevhen-k**
Hi @truskovskiyk 
Here is the code for model training and inference with test for code.

Please, check it.

### CODE REVIEW

**Review by truskovskiyk — COMMENTED** [review]
@yevhen-k looks amazing, could you run this tests in CI so I can see them green?

File: week-3-tests/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+datasets
+transformers
+huggingface_hub
+torch
+scikit-learn
```
Comment (truskovskiyk) [inline week-3-tests/requirements.txt]: please add versions

Comment (yevhen-k) [reply week-3-tests/requirements.txt]: done

**Comment by yevhen-k** [comment]
@truskovskiyk tests are green! please, check it!
after your approve I'll resolve conflicts

**Review by truskovskiyk — APPROVED** [review]
@yevhen-k great work!

**Pull request merged**